recipe/build.sh

```diff
@@ -1,23 +1,67 @@
 #!/bin/bash
 
-# install using pip from the whl files on PyPI
-
-if [ `uname` == Darwin ]; then
-    if [ "$PY_VER" == "2.7" ]; then
-        WHL_FILE=https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-${PKG_VERSION}-py2-none-any.whl
-    else
-        WHL_FILE=https://storage.googleapis.com/tensorflow/mac/cpu/tensorflow-${PKG_VERSION}-py3-none-any.whl
-    fi
-fi
+set -ex
 
-if [ `uname` == Linux ]; then
-    if [ "$PY_VER" == "2.7" ]; then
-        WHL_FILE=https://pypi.org/packages/cp27/t/tensorflow/tensorflow-${PKG_VERSION}-cp27-cp27mu-manylinux1_x86_64.whl
-    elif [ "$PY_VER" == "3.6" ]; then
-        WHL_FILE=https://pypi.org/packages/cp36/t/tensorflow/tensorflow-${PKG_VERSION}-cp36-cp36m-manylinux1_x86_64.whl
-    elif [ "$PY_VER" == "3.7" ]; then
-        WHL_FILE=https://pypi.org/packages/cp37/t/tensorflow/tensorflow-${PKG_VERSION}-cp37-cp37m-manylinux1_x86_64.whl
-    fi
-fi
+if [[ "$target_platform" == linux* ]]; then
+    # do not build with MKL support
+    export TF_NEED_MKL=0
+    export BAZEL_MKL_OPT=""
+
+    mkdir -p ./bazel_output_base
+    export BAZEL_OPTS="--batch "
+
+    # Linux
+    # the following arguments are useful for debugging
+    #    --logging=6
+    #    --subcommands
+
+    # Set compiler and linker flags as bazel does not account for CFLAGS,
+    # CXXFLAGS and LDFLAGS.
+    BUILD_OPTS="
+    --copt=-march=nocona
+    --copt=-mtune=haswell
+    --copt=-ftree-vectorize
+    --copt=-fPIC
+    --copt=-fstack-protector-strong
+    --copt=-O2
+    --cxxopt=-fvisibility-inlines-hidden
+    --cxxopt=-fmessage-length=0
+    --linkopt=-zrelro
+    --linkopt=-znow
+    --verbose_failures
+    ${BAZEL_MKL_OPT}
+    --config=opt"
+    export TF_ENABLE_XLA=1
+
+    # Python settings
+    export PYTHON_BIN_PATH=${PYTHON}
+    export PYTHON_LIB_PATH=${SP_DIR}
+    export USE_DEFAULT_PYTHON_LIB_PATH=1
 
-pip install --no-deps $WHL_FILE
+    # additional settings
+    export CC_OPT_FLAGS="-march=nocona -mtune=haswell"
+    export TF_NEED_IGNITE=1
+    export TF_NEED_OPENCL=0
+    export TF_NEED_OPENCL_SYCL=0
+    export TF_NEED_CUDA=0
+    export TF_NEED_ROCM=0
+    export TF_NEED_MPI=0
+    export TF_DOWNLOAD_CLANG=0
+    export TF_SET_ANDROID_WORKSPACE=0
+    ./configure
+
+    # build using bazel
+    bazel ${BAZEL_OPTS} build ${BUILD_OPTS} //tensorflow/tools/pip_package:build_pip_package
+
+    # build a whl file
+    mkdir -p $SRC_DIR/tensorflow_pkg
+    bazel-bin/tensorflow/tools/pip_package/build_pip_package $SRC_DIR/tensorflow_pkg
+
+    # install the whl using pip
+    pip install --no-deps $SRC_DIR/tensorflow_pkg/*.whl
+
+    # The tensorboard package has the proper entrypoint
+    rm -f ${PREFIX}/bin/tensorboard
+else
+  $PYTHON -m pip install --no-deps -vv $fn
```

Comment:
This does not work :-(
Looks like the variable for the name is not available here.
Also, if I try to use a `script` section in the recipe conda-build complains that I have a build script and a script line. Not sure what is the best way to fix this.